emma-language/src/test/scala/org/emmalanguage/compiler/lang/core/CSESpec.scala

```diff
@@ -30,7 +30,7 @@ class CSESpec extends BaseCompilerSpec {
   val csePipeline: u.Expr[Any] => u.Tree =
     pipeline(typeCheck = true)(
       Core.lnf,
-      tree => time(Core.cse(tree), "cse")
+      TreeTransform("CSESpec.csePipeline/Core.cse", tree => time(Core.cse(tree), "cse"))
```

Comment:
Add a method `timed` to the `TreeTransform` class that copies the logic from `time` using the enclosing `name`.

Then calls

```scala
tree => time(Core.cse(tree), "cse")
```

can be rewritten as follows

```
Core.cse.timed
```

Reply:
Actually, forget the above comment, just rewrite 

```scala
tree => time(Core.cse(tree), "cse")
```

as 

```scala
tree => time(Core.cse(tree), Core.cse.name)
```